controller/src/beerocks/master/db/db.h

```diff
@@ -759,20 +762,20 @@ class db {
      * @brief Set the client's selected-bands.
      * 
      * @param mac MAC address of a client.
-     * @param selected_bands Client selected band/bands. FREQ_UNKNOWN is considered as "not-configured".
+     * @param selected_bands Client selected band/bands. Possible values are bitwise options of eClientSelectedBands.
      * @param save_to_persistent_db If set to true, update the persistent-db (write-through), default is true.
      * @return true on success, otherwise false.
      */
-    bool set_client_selected_bands(const sMacAddr &mac, beerocks::eFreqType selected_bands,
+    bool set_client_selected_bands(const sMacAddr &mac, int8_t selected_bands,
                                    bool save_to_persistent_db = true);
 
     /**
      * @brief Get the client's selected-bands.
      * 
      * @param mac MAC address of a client.
-     * @return Selected band/bands.
+     * @return Selected band/bands. Possible values are bitwise options of eClientSelectedBands.
      */
-    beerocks::eFreqType get_client_selected_bands(const sMacAddr &mac);
+    int8_t get_client_selected_bands(const sMacAddr &mac);
```

Comment:
why not use std::bitmap ?

Reply:
I think you meant std::bitset - so will answer to that.

I could and probably should - thanks.
Will introduce this change as part of the next PR.